OpenBCI Software/03-OpenBCI_Wifi_Server.md

```diff
@@ -50,67 +50,36 @@ Power running to Wifi but NOT Cyton because the `EXT PWR` is in `OFF` position.
 
 If the Cyton is powered up already, the Wifi Shield can not start. Don't power your Cyton and hot plug in a Wifi Shield.
```

Comment:
Not sure if this guide has to be in Software. Has nothing to do with what we have herer